.github/workflows/update_versions.yml

```diff
@@ -0,0 +1,27 @@
+---
+name: Update Versions
+on:
+  schedule:
+  - cron: '0 0 2 1 *' # Run at midnight on January 2nd to pick up Christmas Ruby releases after the holiday break
+jobs:
+  update-versions:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - run: |
+        ./bin/update_version_cache
+        if [ -n "$(git status --porcelain)" ]; then
```

Comment:
This is a cool git command! I've never seen it until now 🌟

Reply:
I stole it from Iryna's PR: https://github.com/Invoca/invoca_logging/pull/64/files :)